prospector/profile.yml

```diff
@@ -8,6 +8,23 @@ pylint:
   disable:
     - broad-except
     - line-too-long
+    - too-many-public-methods
+    - invalid-name
+    - too-many-arguments
+    - too-many-locals
+    - too-few-public-methods
+    - too-many-branches
+    - too-many-statements
+    - redefined-outer-name
+    - too-many-function-args
+    - inconsistent-return-statements
```

Comment:
I would like to see the examples of this one first before disabling. In my opinion, it also should not be there.